build-logic/src/main/kotlin/ktlint-kotlin-common.gradle.kts

```diff
@@ -8,19 +8,32 @@ plugins {
 kotlin {
     // All modules, the CLI included, must have an explicit API
     explicitApi()
+    jvmToolchain(20)
 }
 
+val targetJavaVersion = JavaVersion.VERSION_1_8
+tasks.withType<JavaCompile>().configureEach {
+    options.release.set(targetJavaVersion.majorVersion.toInt())
+}
 tasks.withType<KotlinCompile>().configureEach {
-    compilerOptions {
-        jvmTarget.set(JvmTarget.JVM_1_8)
-    }
+    kotlinOptions.jvmTarget = targetJavaVersion.toString()
```

Comment:
I can keep  `compilerOptions`, AFAIK it doesn't matter. it was easier for me to copy

Reply:
>Should keep compilerOptions, kotlinOptions is deprecated.

Can you link the documentation?

Reply:
https://kotlinlang.org/docs/whatsnew18.html#limitations